blog/content/post/2020-04-26-measuring_memory_usage_rte.md

```diff
@@ -0,0 +1,217 @@
+KiB Mem :   248564 total,    41848 free,    40556 used,   166160 buff/cache
+KiB Swap:   131068 total,   131068 free,        0 used.   168392 avail Mem
+```
+
+`top` output during tests:
+
+```
+load average: 0.53, 0.30, 0.12
+Tasks: 132 total,   1 running, 131 sleeping,   0 stopped,   0 zombie
+%Cpu(s):  0.0 us,  0.0 sy,  0.0 ni, 99.9 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
+KiB Mem :   248564 total,    38644 free,    42900 used,   167020 buff/cache
+KiB Swap:   131068 total,   131068 free,        0 used.   165280 avail Mem
+```
+
+```
+load average: 0.03, 0.13, 0.10
+Tasks: 125 total,   1 running, 124 sleeping,   0 stopped,   0 zombie
+%Cpu(s):  0.2 us,  0.6 sy,  0.0 ni, 99.2 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
+KiB Mem :   248564 total,    44672 free,    36524 used,   167368 buff/cache
+KiB Swap:   131068 total,   131068 free,        0 used.   171404 avail Mem
+```
+
+#### Comparison table
+
+|                       |load   | CPU temp   | Mem usage    |
+|:----------------------|:-----:|:----------:|:------------:|
+|**before tests**       | 0.03  | 43.9&deg;C | 32MB / 242MB |
+|**during tests (max)** | 0.53  | 51&deg;C   | 65MB / 242MB |
+|**after tests**        | 0.12  | 44.2&deg;C | 33MB / 242MB |
+
+### Yocto
+`free` output before tests:
+
+```
+total  used  free  shared  buff/cache   available
+Mem:    244    31   140       9          72         195
+Swap:     0     0     0
+```
+
+`free` output during tests:
+
+```
+total  used  free  shared  buff/cache   available
+Mem:    244    35   139       9          72         194
+Swap:     0     0     0
+```
+
+`top` output before tests:
+
+```
+load average: 0.01, 0.03, 0.01
+Tasks:  88 total,   1 running,  47 sleeping,   0 stopped,   0 zombie
+%Cpu(s):  0.3 us,  0.8 sy,  0.0 ni, 98.8 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
+KiB Mem :   250680 total,   144612 free,    31752 used,    74316 buff/cache
+KiB Swap:        0 total,        0 free,        0 used.   200956 avail Mem
+```
+
+`top` output during tests:
+
+```
+load average: 0.05, 0.04, 0.01
+Tasks:  90 total,   1 running,  49 sleeping,   0 stopped,   0 zombie
+%Cpu(s):  0.2 us,  0.9 sy,  0.0 ni, 98.8 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
+KiB Mem :   250680 total,   142676 free,    33552 used,    74452 buff/cache
+KiB Swap:        0 total,        0 free,        0 used.   199148 avail Mem
+```
+
+```
+load average: 0.08, 0.12, 0.06
+Tasks:  90 total,   1 running,  49 sleeping,   0 stopped,   0 zombie
+%Cpu(s):  4.1 us,  8.9 sy,  0.0 ni, 87.0 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
+KiB Mem :   250680 total,   144328 free,    32124 used,    74228 buff/cache
+KiB Swap:        0 total,        0 free,        0 used.   200580 avail Mem
+```
+
+#### Comparison table
+
+|                        | load  | Mem usage    |
+|:-----------------------|:-----:|:------------:|
+|**before tests**        | 0.03  | 31MB / 244MB |
+|**during tests (max)**  | 0.12  | 35MB / 244MB |
+|**after tests**         | 0.04  | 31MB / 244MB |
+
+
+## Summary
```

Comment:
@pkonkol You could write something more about what is the origin of these differences between Yocto and Armbian.